template/README.md.jinja

```diff
@@ -1,21 +1,19 @@
-| Meta | [![{{ license }}](https://img.shields.io/pypi/l/{{project_slug}}.svg)](LICENSE) [![Code of Conduct](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](.github/CODE_OF_CONDUCT.md) [![Checked with mypy](https://www.mypy-lang.org/static/mypy_badge.svg)](https://mypy-lang.org/) [![Code Style Black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black) [![Linting: Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/charliermarsh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff) |
-| Automation |                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+[![Latest PyPI Version](https://img.shields.io/pypi/v/{{project_slug}}.svg)](https://pypi.org/project/{{project_slug}}/) [![Supported Python Versions](https://img.shields.io/pypi/pyversions/{{project_slug}}.svg)](https://pypi.org/project/{{project_slug}}/) [![Documentation](https://readthedocs.org/projects/{{project_slug}}/badge/?version=latest)](https://{{project_slug}}.readthedocs.io/en/latest/?badge=latest)
+
+[![{{ license }}](https://img.shields.io/pypi/l/{{project_slug}}.svg)](LICENSE) [![Code of Conduct](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](.github/CODE_OF_CONDUCT.md) [![Checked with mypy](https://www.mypy-lang.org/static/mypy_badge.svg)](https://mypy-lang.org/) [![Code Style Black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black) [![Linting: Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/charliermarsh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
I find that a table really improves the legibility of badges. I agree that if you don't have the right plugins/extensions, they are awkward to maintain.

Reply:
My main concern is how it renders on GitHub.

Reply:
Like this, for example, https://github.com/taxprofiler/taxpasta/. Looks fine to me at least.

Reply:
Ah, I think the problem then is that there is nothing currently in the Automation row.